ubuntu22.04/Dockerfile

```diff
@@ -85,11 +85,14 @@ RUN if [ "$DRIVER_TYPE" != "vgpu" ] && [ "$TARGETARCH" != "arm64" ]; then \
     apt-get update && \
     apt-get install -y --no-install-recommends nvidia-fabricmanager-${DRIVER_BRANCH}=${DRIVER_VERSION}-1 \
     libnvidia-nscq-${DRIVER_BRANCH}=${DRIVER_VERSION}-1 && \
+    if [ "$DRIVER_BRANCH" -ge "570" ]; then \
+    apt-get install -y --no-install-recommends libnvsdm-${DRIVER_BRANCH}=${DRIVER_VERSION}-1; \
+    fi && \
     rm -rf /var/lib/apt/lists/*; fi
 
 RUN if [ "$DRIVER_TYPE" != "vgpu" ] && [ "$DRIVER_BRANCH" -ge "550" ]; then \
     apt-get update && \
-    apt-get install -y --no-install-recommends nvidia-imex-${DRIVER_BRANCH}=${DRIVER_VERSION}-1 && \
+    apt-get install -y --no-install-recommends nvidia-imex-${DRIVER_BRANCH}=${DRIVER_VERSION}-1 nvlsm && \
```

Comment:
Why is the installation of nvlsm and libnvsdm separated?

Reply:
I believe gb200 needs nvlsm. So it can't be skipped for arm unlike libnvsdm

Reply:
Since FM is supported on arm, maybe its time to start installing FM in our ARM based containers? That would simplify the changes here and allow us to install all of these additional packages in the same conditional block.

Reply:
Sure, that sounds good to me